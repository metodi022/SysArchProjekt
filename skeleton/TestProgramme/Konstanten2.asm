lui $10, 0x34545 
ori $10, 0x56700
ori $10, 0x34545
sw $10, 4($0)
