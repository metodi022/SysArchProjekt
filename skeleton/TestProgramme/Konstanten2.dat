0x3c013456
0x34211234
0x34211111
0xac010004
