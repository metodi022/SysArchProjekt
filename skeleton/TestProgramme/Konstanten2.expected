@1 34561335
